### ISSUE TITLE
[Critical] Action cannot be performed with a screen reader turned on

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#91]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87539 |
| **Checkpoint** | 2.1 Keyboard Accessible › Keyboard Navigation (2.1.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 04 Work Items Workflow |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/work_items> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/05/26 at 2:36 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

Action cannot be performed with a screen reader turned on.
The Remove Filter buttons for Action and Status cannot be activated with the screen reader turned on. See Work Items User Experience Video at 12:45.
Also occurs on 03 Objects page.

## Recommendation

Rule : 
Keyboard functionality MUST not break when a screen reader is turned on.

How To Fix: 
Fix this issue by ensuring that keyboard functionality does not stop working when a screen reader is turned on. 

The best way to do this is to use native HTML elements when HTML equivalents exist (e.g. anchor, button, select, text area, and input elements). 

When custom (non native HTML) controls have to be used, you must ensure they can be focused with a keyboard (often using tabindex="0") and operated with a keyboard (using device-independent JavaScript event handlers). 

The ARIA Authoring Practices Guide also provides information on expected keyboard interaction patterns for custom controls and widgets and working examples (see References below).

Reference: 
ARIA Authoring Practices Guide Patterns: https://www.w3.org/WAI/ARIA/apg/patterns/

Background: 
Sometimes keyboard interactions are coded in such a way that keyboard functionality is broken when a screen reader is turned on. Most often this occurs when non-native HTML elements are used and proper ARIA and JavaScript event handler techniques are not followed. If you are using non-native HTML elements, it is advised to check the keyboard functionality with just a keyboard and also with a screen reader (NVDA, JAWS, VoiceOver) turned on.

## Source Code

```
N/A
```

## Screenshots

![04_keyboard nav](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87539/04_keyboard_nav.png)
